## Controller Params
tf: 10 # Final Time
N: 50 # Number of segments
Nk: 9 # Number of knots for parametrization
Ns: 240 #Number of samples

#x,y,theta,v
#x0:   [0.0, 0.0, 0.0, 0.0]   #Initial State
x0:   [0.0, 1.5, -1.57, 0.0]   #Initial State
xf:   [6.0, 3.0, -3.14, 0.0] #Final State
Qf:   [10.0, 10.0, 5.0, 1.0] #Final Cost
Q:    [0.0, 0.0, 0.0, 1.0]   #State Cost
#Q:    [0.0, 0.0, 0.0, 0.0]   #State Cost

#desired velocity, steering angle
R:    [0.01, 0.5] #Control Cost

# GN Based feedback params
#x0bar: [0.0, 1.0, -1.57, 0.0]   #Initial State
x0bar: [0.0, 0.0, 0.0, 0.0]   #Initial State
tf_gn: 1 # Final Time in GN
